docs/concepts-broker-load-balancing-types.md

```diff
@@ -6,55 +6,38 @@ sidebar_label: "Types"
-Unload balance (1/2)| Verify if the extensible broker load balancer can decrease the load balance time if some brokers join or leave a cluster.| Measure the time to balance the load (10,000 topics/publishers, 100 bundles) from scaling up from 5 brokers to 10 brokers. |7 mins <br/><br/> The load cannot be balanced without further tuning.|5 mins <br/><br/> The load can be balanced without further tuning.
-Unload balance (2/2)|Same as above|Measure the time to balance the load (10,000 topics/publishers, 100 bundles) from scaling down from 10 brokers to 5 brokers. | 5 mins | 3 mins
-Split balance|Verify if the extensible broker load balancer can decrease the time of automatic bundle splitting and balance the overloaded topics.| Measure the time of splitting 1 bundle into 128 bundles and balancing the load (10,000 topics/publishers) when hot-spot topics have high loads.|15 mins|13 mins
+| Dimension                                  | Modular broker load balancer                                                                                                                                                                                                                                                                  |Extensible broker load balancer
+|--------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---
+ Available version                          | Pulsar 1.7.0 <br/> <br/>2017                                                                                                                                                                                                                                                                  |Pulsar 3.0.0 <br/> <br/>2023
+ Load Balance Metadata Store and Replication | Dependent on ZooKeeper for load balance metadata store and replication.<br/> <br/> - All broker load data and all bundle load data are stored in ZooKeeper and replicated to all brokers via ZooKeeper watchers. <br/><br/>- All bundle ownerships are stored in Zookeeper as ephemeral locks. | Dependent on system topics and table views for load balance metadata store and replication. <br/> <br/> - All broker load data are stored in a non-persistent system topic and replicated to all brokers via table views. <br/> <br/>- Each broker publishes only top k bundles' load data in a non-persistent system topic, and only the leader broker consumes it via a table view. <br/> <br/> - Bundle ownerships are stored in a persistent system topic and replicated to all brokers via table views.<br/> <br/> * The absolute size of the replicated load data and the complexity of the replication decreased due to passing the data through memory-only, non-persistent topics.
```

Comment:
```suggestion
 Load Balance Metadata Store and Replication | Dependent on ZooKeeper for load balance metadata store and replication.<br/> <br/> - All broker load data and all bundle load data are stored in ZooKeeper and replicated to all brokers via ZooKeeper watchers. <br/><br/>- All bundle ownerships are stored in ZooKeeper as ephemeral locks. | Dependent on system topics and table views for load balance metadata store and replication. <br/> <br/> - All broker load data are stored in non-persistent system topics and replicated to all brokers via table views. <br/> <br/>- Each broker publishes only top k bundles' load data in non-persistent system topics, and only the leader broker consumes it via table views. <br/> <br/> - Bundle ownerships are stored in persistent system topics and replicated to all brokers via table views.<br/> <br/> - The absolute size of the replicated load data and the complexity of the replication decreased due to passing the data through memory-only, non-persistent topics.
```

Reply:
The broker load data is stored in a single non-persistent topic. (1st topic)
The topk bundle load data is stored in a separate non-persistent topic.(2nd topic)
Bundle ownerships are stored in a separate persistent topic.(3rd topic)

I intentionally used `*` to note that the following is an annotation.

`the absolute size of the replicated load data and the complexity of the replication decreased due to passing the data through memory-only, non-persistent topics`

Reply:
Suggest using another format (like `Note: xxxx`), otherwise, users might take it as an item of the unordered list. 

<img width="783" alt="image" src="https://github.com/apache/pulsar-site/assets/50226895/9cffc9ed-2b70-4811-a624-825f0572808b">

Reply:
Replcated `*` with `Note`.